packages/patternlab/styleguide/source/_patterns/05-pages/service.json

```diff
@@ -84,7 +84,7 @@
     "keyActions": {
       "compHeading": {
         "title": "First time?",
-        "sub": true
+        "sub": false

```

Comment:
can we clean up `sub`? It's not documented, and it's been replaced by level.

Reply:
`sub` has been replaced with `level` with a level number value.